tests/k8st/dind-cluster.sh

```diff
@@ -0,0 +1,2336 @@
+#!/bin/bash
```

Comment:
LOL @neiljerram told me to do exactly that to avoid things in our build changing under our feet.

And no, its not an exact copy of the script from the repo you reference above.  

OK - in that case the other possibility is to fork the kubeadm-dind-cluster repo into tigera, make our changes and then reference that here.

Reply:
Either way, we're *not* using the script from upstream unmodified.  

So we need to fork `kubeadm-dind-cluster` somewhere.  And it ought to be into this org, no?

Reply:
@neiljerram I think we can remove this checked-in copy now.